drivers/SmartThings/philips-hue/src/stray_device_helper.lua

```diff
@@ -103,114 +103,112 @@ function StrayDeviceHelper.spawn()
           found_bridges[msg_device.id] = msg.device
           local bridge_device_uuid = msg_device.id
 
-          -- TODO: We can optimize around this by keeping track of whether or not this bridge
-          -- needs to be scanned (maybe skip scanning if there are no stray devices?)
-          --
-          -- @doug.stephen@smartthings.com
-          log.info(
-            string.format(
-              "Stray devices handler notified of new bridge %s, scanning bridge",
-              (msg.device.label or msg.device.device_network_id or msg.device.id or "unknown bridge")
+          if next(stray_devices) ~= nil then
+            log.info(
```

Comment:
Do you want a log for the else case to know if this check happened, but there were no stray devices?

Reply:
This whole thing needs to be refactored still, but it's low priority because it can typically only get hit during migration. There are very rare edge cases where it can still happen outside of migration, but it would involve inventory corruption.

TL;DR probably, yes, but the stray resolver needs a deeper refactor in general that shouldn't block SVT.